gspread/worksheet.py

```diff
@@ -3201,3 +3204,88 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        range: str,
+        condition_type: ValidationConditionType,
+        values: Iterable[Any],
+        inputMessage: Optional[str] = None,
+        strict: bool = False,
+        showCustomUi: bool = False,
+    ) -> Any:
+        """Adds a data validation rule to any given range.
+
+        .. note::
+
+            ``condition_type`` values are explained here: `ConditionType`_
+
+            .. _ConditionType: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/other#ConditionType
+
+
+        :param str source: The A1 notation of the source range to move
+        :param condition_type: The sort of condition to apply.
+        :param values: List of condition values.
+        :type values: Any
+        :param str inputMessage: Message to show for the validation.
+        :param bool strict: Whether to reject invalid data or not.
+        :param bool showCustomUi: Whether to show a custom UI(Dropdown) for list values.
+
+        **Examples**
+
+        .. code-block:: python
+
+            import gspread
+            from gspread.utils import ValidationConditionType
+
+
+            ...
+
+            ws = spreadsheet.sheet1
+
+            ws.add_validation(
+                'A1',
+                ValidationConditionType.number_greater,
+                10,
+                strict=True,
+                inputMessage='Value must be greater than 10',
+            )
+
+            ws.add_validation(
+                'C2:C7',
+                ValidationConditionType.one_of_list,
+                'Yes',
+                'No',
```

Comment:
does this example work with the new format of [values]?

Reply:
good catch  ! no it does not in fact, values must be put into a single list.